### DISCUSSION TITLE
How can I fix this error?: No such module "node:fs". I'm following the CF docs, what I'm doing wrong?

### DISCUSSION THREAD

**Discussion by ghost** [cloudflare/workers-sdk#5776] [Q&A]
Hi community!

I'm using NodeJS v22.0.0, Hono JS and Cloudflare Workers and I'm having an issue with "fs" and "path" modules:
```
import { Hono } from 'hono'
import { serveStatic } from 'hono/cloudflare-workers'
import fs from 'node:fs'
import path from 'node:path'
```

But when I run `wrangler dev src/index.ts` it raise an error:
```
X [ERROR] service core:user:example: Uncaught Error: No such module "node:fs".

    imported from "index.js"

X [ERROR] MiniflareCoreError [ERR_RUNTIME_FAILURE]: The Workers runtime failed to start. There is likely additional logging output above.
```
Reading the docs I need to setup my `wrangler.toml` to be compatible with Node adding `compatibility_flags = [ "nodejs_compat" ]` to the file, but I already have it:
```
name = "example"
main = "src/index.ts"
compatibility_date = "2023-12-01"
compatibility_flags = [ "nodejs_compat" ]
```

My tsconfig.json is configured to recognize NodeJS and Cloudflare Workers types, that's why VSCode doesn't recognize any error, it only shows up when running the wrangler dev src/index.ts:
```
{
    "compilerOptions": {
        "target": "ESNext",
        "module": "ESNext",
        "moduleResolution": "Bundler",
        "strict": true,
        "skipLibCheck": true,
        "lib": ["ESNext"],
        "types": ["@types/node", "@cloudflare/workers-types"],
        "jsx": "react-jsx",
        "jsxImportSource": "hono/jsx"
    }
}
```

**Comment by treeder** [comment]
You should probably try to avoid using node libs. What are you trying to do?
